### DISCUSSION TITLE
If I buy a license will I have a problem with the connection ?

### DISCUSSION THREAD

**Discussion by luapfr** [rustdesk/rustdesk#7759] [General]
Today here in Brazil the free version is unable to contact each other. Does this also happen if I have a Pro license ?

**Comment by rustdesk** [comment]
The Pro license is for self-hosting, we do not have cloud option yet.

**Comment by luapfr** [comment]
> The Pro license is for self-hosting, we do not have cloud option yet.

But I don't understand.

For two days I was able to access the free version.

Why did it stop working today ?

It's not clear to me if even if I buy a 10-user license I still need to host my own server?

I don't have a computer that stays on 24 hours a day for my two technicians to support my clients, so I want to buy a license where I will always have access, just like teamviewer and anyDesk

It seems that you encourage the creation of a RustDesk Hosting Connection Server, which for me is unfeasible.

If you could explain it to me in more detail I would be very grateful.
